workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectInitializationSystem.cs

```diff
@@ -114,7 +96,8 @@ protected override void OnUpdate()
                 }
 
                 entityGameObjectCache.Remove(entity);
-                UnityObjectDestroyer.Destroy(gameObject);
+                gameObjectCreator.OnEntityGameObjectRemoved(new SpatialOSEntity(entity, EntityManager),
+                    worker, gameObject);
```

Comment:
it looks like the `OnDestroyManager` part was removed. Could you add this back in.

Reply:
That was in a different PR and I've not yet rebased / merged. Or possibly Jonas's PR which is not yet merged anyway? Either way it's not forgotten, just somewhere else.